atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraServersConfigsTest.java

```diff
@@ -60,17 +61,30 @@
     }
 
     @Test
-    public void canDeserializeMultiEntryDefault() throws IOException {
+    public void testCanDeserializeMultiEntryDefault() throws IOException {
         assertThat(deserializeClassFromFile("testServersConfigDefaultMulti.yml"))
                 .isEqualTo(defaultConfig(THRIFT_SERVER_1, THRIFT_SERVER_2));
     }
 
     @Test
-    public void canDeserializeMultiEntryCqlCapable() throws IOException {
+    public void testCanDeserializeMultiEntryCqlCapable() throws IOException {
         assertThat(deserializeClassFromFile("testServersConfigCqlCapableMulti.yml"))
                 .isEqualTo(CQL_CAPABLE_CONFIG);
     }
 
+    @Test
+    public void testHostAreSame() throws IOException {
+        CqlCapableConfig cqlCapableConfig =
+                (CqlCapableConfig) deserializeClassFromFile("testServersConfigCqlCapableMulti.yml");
```

Comment:
Casting seems odd, why not just pass the class in whilst deserialising instead of deserialising to the more general type?